include/nbl/builtin/hlsl/ieee754.hlsl

```diff
@@ -0,0 +1,128 @@
+#ifndef _NBL_BUILTIN_HLSL_IEE754_H_INCLUDED_
+#define _NBL_BUILTIN_HLSL_IEE754_H_INCLUDED_
+
+#include <nbl/builtin/hlsl/type_traits.hlsl>
+
+namespace nbl::hlsl::ieee754
+{
+	namespace impl
+	{
```

Comment:
1. nested namespace syntax doesn't compile with DXC
2. don't indent extra because of namespaces